### ISSUE TITLE
Problems with MacPro + Sonnet Breakaway 500 + ATI 1080 

### ISSUE THREAD

**Issue by Mistobaan** [yifanlu/NVIDIAEGPUSupport#4]
I followed the instructions, step by steps.

In attachment the Screenshot and the output of AGDCDiagnose.

Hardware:
- MacPro (Late 2013)
```
  Model Name:	Mac Pro
  Model Identifier:	MacPro6,1
  Processor Name:	12-Core Intel Xeon E5
  Processor Speed:	2.7 GHz
  Number of Processors:	1
  Total Number of Cores:	12
  L2 Cache (per Core):	256 KB
  L3 Cache:	30 MB
  Memory:	64 GB
```
- https://www.sonnetstore.com/collections/thunderbolt-expansion-systems/products/egfx-breakaway-box-550
- https://www.asus.com/us/Graphics-Cards/ROG-STRIX-GTX1080-O8G-GAMING/

Let me know if you need more info 

![blank](https://user-images.githubusercontent.com/112599/32149042-44f3a614-bcbc-11e7-8815-6a4ca6fcb5fe.png)
[AGDCDiagnose.txt](https://github.com/yifanlu/NVIDIAEGPUSupport/files/1425541/AGDCDiagnose.txt)

**Comment by yifanlu** [comment]
Can you log out and in again?

**Comment by Mistobaan** [comment]
Logging in and out did the trick. But started flickering until I disconnected all the additional monitors.

**Comment by Mistobaan** [comment]
I am running tensorflow on the egpu and seems like it does not free memory. 
```
2017-10-30 19:53:16.770262: I tensorflow/core/common_runtime/gpu/gpu_device.cc:977] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:51:00.0)
2017-10-30 19:53:16.770670: E tensorflow/stream_executor/cuda/cuda_driver.cc:893] failed to allocate 7.46G (8012093440 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
2017-10-30 19:53:16.770782: E tensorflow/stream_executor/cuda/cuda_driver.cc:893] failed to allocate 6.71G (7210884096 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
```
Is there a limit ?

**Comment by yifanlu** [comment]
Don't ask me, ask nvidia/tensorflow.

**Comment by Mistobaan** [comment]
Would you mind pushing a branch with the support for `AppleGPUWrangler`? maybe that works for my case. In which part of the apple documentation do you read about these new updates?

**Comment by yifanlu** [comment]
No, that's not the problem. I don't know if there's documentation, but I just disassembled the drivers and read the assembly code.

**Comment by Mistobaan** [comment]
Thank you @yifanlu ! 

I am gonna leave this issue open for now in case someone else is trying to run this configuration :
- Mac Pro 
- High Sierra
- Sonnet 550
- Asus GeForce 1080
- CUDA 8.0
and is successful
